test/fixtures/detect-existing-browser-api-object/content.js

```diff
@@ -10,12 +10,27 @@ test("browser api object in content script", (t) => {
     // On Firefox, window is not the global object for content scripts, and so we expect window.browser to not
     // be defined.
     t.equal(window.browser, undefined, "window.browser is expected to be undefined on Firefox");
+
+    try {
```

Comment:
These tests don't really belong to this test. Could you put them in their own location?

Reply:
Same file, different `test()`, right?